types/are.na/index.d.ts

```diff
@@ -0,0 +1,621 @@
+         */
+        create(title: string, status: Arena.ChannelStatus): Promise<void>;
+        /**
+         * Creates a new channel. Called as channel(title).create(status). Title is required, status is optional.
+         */
+        create(status: Arena.ChannelStatus): Promise<void>;
+        /**
+         * Delete the channel. Can be called as channel(slug).delete() or channel().delete(slug).
+         */
+        delete(slug: string): Promise<void>;
+        /**
+         * Update the channel's attributes. params should be an object and can include title and/or status.
+         * Currently it appears that the are.na API requires both values.
+         * If title is not set, an error will occur.
+         * If status is not set, it will default to "public".
+         */
+        update(params: { title?: string; status?: Arena.ChannelStatus }): Promise<Arena.Channel>;
+        /**
+         * Add collaborators to a channel. Pass userIds as an Array or multiple arguments.
+         */
+        addCollaborators(userIDs: string[]): Promise<Arena.User[]>;
+        /**
+         * Remove collaborators from a channel. Accepts userIds in the same format as addCollaborators.
+         */
+        deleteCollaborators(userIDs: string[]): Promise<Arena.User[]>;
+        /**
+         * Create a block and add it to the channel. Specify textual content or a source link.
+         */
+        createBlock(content: any): Promise<Arena.Block>;
+        /**
+         * Remove a block from the channel
+         */
+        deleteBlock(blockID: number): Promise<void>;
+    };
+
+    /**
+     * Base object for block related API requests
+     *
+     * ```js
+     * // Basic usage
+     * arena.block(3088).get()
+     * ```
+     *
+     * @param id - Block id, such as 3088
+     * @param params - Additional parameters
+     */
+    block(
+        id: number,
+        params?: Params,
+    ): {
+        /**
+         * Get the block specified by id
+         */
+        get(params?: Params): Promise<Arena.Block>;
+        /**
+         * Get a list of the channels a block belongs to
+         */
+        channels(params?: Params): Promise<Arena.Channel[]>;
+        /**
+         * Create a block and add it to the channel.
+         * Specify textual content or a source link.
+         */
+        create(channelSlug: string, content: any): Promise<Arena.Channel>;
+        /**
+         * Update a block. Pass an object with one or more of content,
+         * title or description fields to update those fields.
+         */
+        update(params: { content?: any; title?: string; description?: string }): Promise<void>;
+    };
+
+    /**
+     * Base object for user related API requests
+     *
+     * ```js
+     * // Basic usage
+     * arena.user(16).get()
+     * ```
+     *
+     * @param id - User id, such as 16
+     * @param params - Additional parameters
+     */
+    user(
+        id: number,
```

Comment:
`id` is string
https://github.com/ivangreene/arena-js/blob/master/index.js#L153

Reply:
Good catch - I have tested and these functions will work with either string or number and it makes more ergonomic sense as a number. I've changed this and the `id` cases above to support `string | number` to be more accurate